resources/scss/componets/02_typography.scss

```diff
@@ -0,0 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Rubik&display=swap');
```

Comment:
Move this font from here and set it in **resources/views/layouts/components/assets.blade.php**

Reply:
remove this code and provide link of google font in assets.blade.php